packages/xod-client/src/project/actions.js

```diff
@@ -65,6 +72,50 @@ export const openWorkspace = libs => ({
   payload: libs,
 });
 
+export const requestPublishProject = () => (dispatch, getState) => {
+  const isAuthorised = foldMaybe(false, R.T, getGrant(getState()));
+
+  if (!isAuthorised) {
+    dispatch(addError(LOG_IN_TO_CONTINUE));
+    // TODO: open account pane?
```

Comment:
I think it's a good idea.
Why it's a TODO? Is it more complex than dispatch another one action?